CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ Changes that are not related to specific components
 #### Fixed
 
 - [Component] What bugs/typos are fixed?
+- [Header.LanguageSelector] Added `aria-current` to currently selected language button
```

Comment:
copy-pasted from different PR, not related to this one?

Reply:
Might be that it got there when I did a rebase. Now it's removed.